skimage/feature/texture.py

```diff
@@ -264,6 +274,19 @@ def graycoprops(P, prop='contrast'):
         results = np.sqrt(asm)
     elif prop == 'ASM':
         results = np.sum(P**2, axis=(0, 1))
+    elif prop == 'mean':
+        _, results = glcm_mean()
+    elif prop == 'variance':
+        I, mean = glcm_mean()
+        results = np.sum(P * ((I - mean) ** 2), axis=(0, 1))
+    elif prop == 'std':
+        I, mean = glcm_mean()
+        var = np.sum(P * ((I - mean) ** 2), axis=(0, 1))
+        results = np.sqrt(var)
+    elif prop == 'entropy':
+        ln = -np.log(P, where=(P != 0), out=np.zeros_like(P))
```

Comment:
```suggestion
        ln = -np.log(P, where=(P != 0))
```
Reading the [docs](https://numpy.org/doc/stable/reference/generated/numpy.log.html), this is useless / not the intended use. If you wanted to specify `out`, you could do:

```py
ln = np.zeros_like(P)
-np.log(P, where=(P != 0), out=ln)
```

If you don't specify `out`, "a freshly-allocated array is returned" anyway.

Reply:
@lagru suggested using the `np.zeros_like()` function to initialize the output in order to avoid undesirable behaviors, as the documentation states that when **where** is used, the locations where the condition is not met are left uninitialized, which is not necessarily 0 (expected value).

> This condition is broadcast over the input. At locations where the condition is True, the out array will be set to the ufunc result. Elsewhere, the out array will retain its original value. Note that if an uninitialized out array is created via the default out=None, locations within it where the condition is False will remain uninitialized.

Reply:
Ok. 
Thank you :slightly_smiling_face:.